test/common/cdp.py

```diff
@@ -170,12 +171,15 @@ def browser_cmd(self, cdp_port, env):
         if not exe:
             raise SystemError(self.browser + " is not installed")
 
+        ws = "1920x1200"
```

Comment:
on line 204 we set size of firefox browser. I would say that setting `TEST_MOBILE` env should work for both (even if we don't have context for it right now)

Reply:
Fixed.